app/model/GroupUser.php

```
@@ -0,0 +1,85 @@
+     * @return array
+     */
+    public function getMembers($groupId)
+    {
+        return $this->findBy(['group_id' => $groupId])->fetchAll();
+    }
+
+    /**
+     * @param int $groupId
+     * @param bool $active
+     */
+    public function attend($groupId, $active = true)
```

PHPMD (2.15.0)

[error] The method attend has a boolean flag argument $active, which is a certain sign of a Single Responsibility Principle violation. (undefined)

(BooleanArgumentFlag)